cg/meta/create_validation_cases/validation_cases_api.py

```diff
@@ -0,0 +1,187 @@
+"""Module that holds the api to create validation cases."""
+
+import logging
+from pathlib import Path
+
+from housekeeper.store.models import File, Version
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.constants import SequencingFileTag
+from cg.meta.create_validation_cases.validation_case_data import ValidationCaseData
+from cg.meta.create_validation_cases.validation_data_input import ValidationDataInput
+from cg.store.models import Case, CaseSample, Sample
+from cg.store.store import Store
+from cg.utils.files import copy_file, get_files_matching_pattern, rename_file
+
+LOG = logging.getLogger(__name__)
+
+
+class CreateValidationCaseAPI:
+
+    def __init__(self, status_db: Store, housekeeper_api: HousekeeperAPI, dry_run: bool = False):
+        self.status_db: Store = status_db
+        self.hk_api: HousekeeperAPI = housekeeper_api
+        self.dry_run = dry_run
+
+    def get_validation_case_data(self, validation_data_input: ValidationDataInput):
+        return ValidationCaseData(
+            status_db=self.status_db,
+            hk_api=self.hk_api,
+            validation_data_input=validation_data_input,
+        )
+
+    def store_validation_samples(self, validation_case_data: ValidationCaseData) -> None:
```

Comment:
should this also store info in LIMS?

Reply:
What information is pulled from LIMS

Reply:
I would say mainly the data that we have mocked in the `MockLimsAPI`

Reply:
decided not to store anything in LIMS. Information retrieval from lims will be handled during validation call.